src/pages/journal/journal-components/journal-test-outcome/journal-test-outcome.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+@Component({
+  selector: 'journal-test-outcome',
+  templateUrl: 'journal-test-outcome.html'
+})
+export class JournalTestOutcomeComponent implements OnInit {
+  @Input()
+  slot: any;
+
+  @Output() onStartTest = new EventEmitter<any>();
```

Comment:
Think we need to have a chat as a team about using Angular's "Output Event Emitters" as this is effectively creating & registering new state in the application which we should be using `ngrx` for. More info here - https://wiki.i-env.net/display/MES/State+Management on how to do this.

From previous experience, Output event emitters very quickly and easily start to get unwieldy. As there's no central governance around what they're doing (any component can emit anything it wants), they become a nightmare to manage when scaling up.